src/daemon/daemon.cpp

```diff
@@ -928,6 +928,13 @@ try // clang-format on
                     alias_entry->set_alias(alias.toStdString());
                 }
 
+                // If no aliases are found, then it's an invalid entry
+                if (entry->aliases_info().empty())
+                {
+                    response.mutable_images_info()->RemoveLast();
+                    continue;
+                }
+
```

Comment:
Hmm this feels like a workaround, shouldn't we be not getting invalid entries here?

Reply:
Above this code in the `for` loop is where the check is for if an alias is supported or not.  If no aliases are found, then as this PR proposes, we should not have an entry for the response.  This whole thing is kind of a hack since we added the whole notion of which aliases are supported and which are not depending on the platform well after how we initially implemented remote image querying.  If we want to move all of the logic to determine that somewhere else, then it will require quite a bit of refactoring which beyond the scope of this PR.  This PR simply fixes a crash on Windows and Mac when issuing `multipass find release:`.

Reply:
Well, yeah, I just mean in the `for` above, we shouldn't even be adding an entry unless we know it's not going to be empty?